spec/models/archived_petition_spec.rb

```diff
@@ -25,7 +21,7 @@
     end
 
     it "sorts the results by the created_at timestamp" do
-      expect(ArchivedPetition.search(q: "Petition")).to eq([petition_2, petition_1])
+      expect(ArchivedPetition.search(q: "Petition")).to match_array([petition_2, petition_1])
```

Comment:
Yes, it is, unfortunately I wasn't aware of that - I was using it to force `to_ary` to be called. I'll just have to change it back and call `to_a` manually.